OutOfSchool/OutOfSchool.WebApi.Tests/QuartzJobs/AverageRatingQuartzJobTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.BackgroundJobs.Jobs;
+using OutOfSchool.BusinessLogic.Services.AverageRatings;
+using Quartz;
+
+namespace OutOfSchool.WebApi.Tests.QuartzJobs;
+
+[TestFixture]
+public class AverageRatingQuartzJobTests
+{
+    private readonly Mock<IAverageRatingService> averageRatingServiceMock = new();
+    private readonly Mock<ILogger<AverageRatingQuartzJob>> loggerMock = new();
+    private readonly Mock<IJobExecutionContext> jobExecutionContextMock = new();
```

Comment:
(Test setup) This approach (mocks creation in a ctor) would work good only for a case of 1 test. For more than 1 test run in parallel you could have shared state and concurrency artifacts. Please consider redesign

Reply:
Same for other test classes